harness/determined/common/api/errors.py

```diff
@@ -37,10 +37,15 @@ def __init__(self, response: requests.Response) -> None:
         self.status_code = response.status_code
 
 
+class NotFoundException(APIException):
+    pass
+
+
 class UnauthenticatedException(BadRequestException):
     def __init__(self, username: str):
         super().__init__(
-            message="Unauthenticated: Please use 'det user login <username>' to log in"
+            message="Unauthenticated: Please use 'det user login <username>' for password login or"
+            " 'det auth login --provider=<provider>' to login through an SSO provider"
```

Comment:
maybe put like "or for Enterprise users logging in with an SSO provider, use ... " so oss users dont misparse this